pandas/tests/indexes/period/test_period_range.py

```diff
@@ -214,18 +212,26 @@ def test_uppercase_freq_deprecated_from_time_series(self, freq_depr):
         with tm.assert_produces_warning(FutureWarning, match=msg):
             period_range("2020-01-01 00:00:00 00:00", periods=2, freq=freq_depr)
 
-    @pytest.mark.parametrize("freq_depr", ["2m", "2q-sep", "2y", "2w"])
-    def test_lowercase_freq_deprecated_from_time_series(self, freq_depr):
+    @pytest.mark.parametrize("freq", ["2m", "2q-sep", "2y"])
+    def test_lowercase_freq_from_time_series_raises(self, freq):
         # GH#52536, GH#54939
-        msg = f"'{freq_depr[1:]}' is deprecated and will be removed in a "
-        f"future version. Please use '{freq_depr.upper()[1:]}' instead."
+        msg = f"Invalid frequency: {freq}"
 
-        with tm.assert_produces_warning(FutureWarning, match=msg):
-            period_range(freq=freq_depr, start="1/1/2001", end="12/1/2009")
+        with pytest.raises(ValueError, match=msg):
+            period_range(freq=freq, start="1/1/2001", end="12/1/2009")
 
     @pytest.mark.parametrize("freq", ["2A", "2a", "2A-AUG", "2A-aug"])
     def test_A_raises_from_time_series(self, freq):
         msg = f"Invalid frequency: {freq}"
 
         with pytest.raises(ValueError, match=msg):
             period_range(freq=freq, start="1/1/2001", end="12/1/2009")
+
+    @pytest.mark.parametrize("freq", ["2w"])
+    def test_lowercase_freq_from_time_series_deprecated(self, freq):
+        # GH#52536, GH#54939
+        msg = f"'{freq[1:]}' is deprecated and will be removed in a "
+        f"future version. Please use '{freq.upper()[1:]}' instead."
+
+        with tm.assert_produces_warning(FutureWarning, match=msg):
```

Comment:
Will this deprecation be enforced by 3.0 as well?

Reply:
I am unsure if we should enforce the deprecation of lowercase 'w' by 3.0. Because we want to deprecate the lowercase 'd', 'b', and 'c' frequencies in favor of the uppercase 'D', 'B', and 'C' in 3.0, we can keep 'w' as deprecated along with 'd', 'b', and 'c'.

Do you think it would be better to keep 'w' as deprecated or remove 'w' and deprecate the others?

Reply:
OK I see. If the plan is to also deprecate the lower case aliases in the future we can keep this as is